### ISSUE TITLE
Translation option for text lines

### ISSUE THREAD

**Issue by csBlackWolf** [FriendsOfFlarum/terms#24]
Hello.

I try to make a German Forum with FreeFlarum what has the FoF-Terms extension preinstalled.
So i added my ToS for the registration and now i wanted to translate the Text "I accept the Terms of Service" to German.
I can translate "Terms of Service", but the text before it can't be translated.
I tried to find and change the line with the extension "linguist" but can't find this text line there.

It would be really nice to add a option to translate it without modifying the code, what is not possible if i using FreeFlarum.

With best regards

csBlackWolf

![grafik](https://user-images.githubusercontent.com/30509936/99484195-2a682680-2960-11eb-9419-d40e7683e9c6.png)

**Comment by clarkwinkelmann** [comment]
Hello. I'm not sure I understand which translation you have issues with. There's one named `fof-terms.forum.accept-modal.i-accept` and another named `fof-terms.forum.signup.i-accept` which both contain identical text.

![image](https://user-images.githubusercontent.com/5264300/99587304-33e9a100-29e9-11eb-8dcd-8ba3e88da48c.png)

You need to preserve `<a>{policy}</a>` as it is so it can be substituted with the policy name. If you have a single policy and wanted its name to change between languages, you could hard-code it in that translation in place of `{policy}`.

If you're not seeing the same results as me in the translation search, can you share the version of Linguist and Terms that are installed on that forum? I suppose you can't share `php flarum info` if that's FreeFlarum but you should see the extension version(s) in the Extensions tab.

**Comment by csBlackWolf** [comment]
Hmmm... For me it is not showing, if i type 'i accept' in the search from linguist. 😐
![grafik](https://user-images.githubusercontent.com/30509936/99607373-f9ddc680-2a0b-11eb-9a66-2010c14b8038.png)

**Comment by clarkwinkelmann** [comment]
Yes linguist is only able to list strings that originally exist in at least one of the enabled languages, so this usually requires having the English pack enabled. You can disable the English Pack once you're done translating.

We might be able to show/search English strings for extensions even when English is disabled now that I think of it. Those translations are registered by extensions regardless of the enabled language so we could pull them out.
